src/middlewares/auth.middleware.js

```diff
@@ -32,6 +32,16 @@ const authMiddleware = {
       ResponseService.setError(401, 'No Token supplied');
       return ResponseService.send(res);
     }
+  },
+
+  verifyAdmin: async (req, res, next) => {
```

Comment:
This method name is not descriptive enough and this is similar to feedback that was given on PR #42 , please rename it.

Reply:
I have implemented according @adesege requirements. @gadishimwe any suggestion of how you want me to rename it?

Reply:
I can suggest verifyIfUserIsAdmin

Reply:
@gadishimwe Yes applied